aws/rust-runtime/aws-sigv4/src/http_request/canonical_request.rs

```diff
@@ -736,7 +747,8 @@ mod tests {
             .body("")
             .unwrap();
         let req = SignableRequest::from(&req);
-        let signing_params = signing_params(SigningSettings::default());
+        let identity = Identity::new(Credentials::for_tests(), None);
```

Comment:
lets add `impl Into<Identity>` for credentials. That way you don't have to have this constructor everywhere + it will handle setting the expiry properly.

Reply:
signing params can then accept `impl Into<Identity>`